.github/workflows/publish.yaml

```diff
@@ -59,3 +76,33 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           annotations: ${{ steps.meta.outputs.annotations }}
           push: true
+
+
+  build-and-publish-chart:
```

Comment:
Could this job be moved into the final step of the previous job so it could re-use the existing metadata and login? I think it also needs some kind of check to only run it on new tags. I think it'd currently run on every PR.